loop/routes/rooms.js

```diff
@@ -414,21 +415,30 @@ module.exports = function (apiRouter, conf, logError, storage, auth,
                 sendError(res, 410, errors.EXPIRED, "Participation has expired.");
                 return;
               }
-              res.status(200).json({
-                expires: ttl
+              storage.getRoomParticipant(req.token, participantHmac, function(err, participant) {
+                if (res.serverError(err)) return;
+                req.roomConnectionId = participant.id;
```

Comment:
A small comment expliciting why this is id is replaced by another would be useful (I couldn't follow)